libafl/src/bolts/anymap.rs

```diff
@@ -38,8 +38,6 @@ macro_rules! impl_asany {
     };
 }
 
-// yolo
```

Comment:
It's not a bad easter egg 😁

Reply:
I'm not sure what to do about this one :)

Should I let the comment in the code ?

Reply:
No strong feelings either way